### DISCUSSION TITLE
Support for doorbell dual

### DISCUSSION THREAD

**Discussion by toshibochan** [samemory/homebridge-eufy-security-control#369] [General]
I have new doorbell dual but not recognize for this plug-in. It show “ undefined” in the list.

**Comment by lenoxys** [comment]
> I have new doorbell dual but not recognize for this plug-in. It show “ undefined” in the list.

Hello

You will need to open a case here :

https://github.com/bropat/eufy-security-client

**Reply by toshibochan** [reply]
bropat just added the doorbell. 

https://github.com/bropat/eufy-security-client/releases/tag/2.0.0

**Reply by toshibochan** [reply]
You need to change anything?

**Reply by lenoxys** [reply]
We're no more providing support on this GitHub repo.

We've implemented bropat@2.0.0 on 2.0.0.beta.1.
